src/stash/protos/objects_v30.proto

```diff
@@ -558,13 +558,18 @@ message AuditLogEventV1 {
         StringWrapper database_name = 3;
     }
 
+    message UpdateItemV1 {
```

Comment:
Generally these files should be immutable, and we add a new migration when we need new fields. Is there a compelling reason to make this change?

Reply:
Oh, I see--I had thought that migrations were only necessary to provide interoperability between versions, i.e. in my thinking this wasn't an actual version bump because there's nothing to be done to move between versions. I think the compelling reason to do it this way is because we can and it's less noise. What is the advantage in treating these files as immutable and adding no-op migrations?

Reply:
@ParkMyCar can answer your question, but is out for a few days. I'm not confident in saying whether or not this noop change is safe or not myself. I think your choices are: wait until next week, or add a new version.